demos/client-bundle-example/src/frictionless.html

```diff
@@ -6,24 +6,26 @@
             async function captchaLoaded() {
                 const render = (await import('./assets/procaptcha.bundle')).render
                 console.log('Captcha loaded')
-                render('procaptcha-container', {
+                // Get the Element using elementId
+                const captchaContainer = document.getElementById('procaptcha-container');
+                render(captchaContainer, {
                     siteKey: '5HGjWAeFDfFCWPsjFQdVV2Msvz2XtMktvgocEZcCj68kUMaw',
                     theme: 'dark',
-                    callback: 'onCaptchaVerified',
+                    callback: onCaptchaVerified,
                     captchaType: 'frictionless',
                 })
             }
             window.captchaLoaded = captchaLoaded
 
             // Define a callback function to be called when the CAPTCHA is verified
-            window.onCaptchaVerified = function (output) {
+            function onCaptchaVerified(output) {
                 console.log('Captcha verified, output: ' + JSON.stringify(output))
             }
         </script>
         <script
             id="procaptchaScript"
             type="module"
-            src="./procaptcha.bundle.js?onload=captchaLoaded&render=explicit"
+            src="./assets/procaptcha.bundle.js?onload=captchaLoaded&render=explicit"
```

Comment:
have you changed where the bundle lives? if so, we'll need to change other scripts to output to that dir

Reply:
Should I change this to origin again?